checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -15,6 +32,61 @@
 ENTROPY_KEYWORD_COMBINATOR_LIMIT = 3
 ENTROPY_KEYWORD_LIMIT = 4.5
 
+INDENTATION_PATTERN = re.compile(r'(^[\r\t\f\v ]*(?:-?[\r\t\f\v ]+)?)')
+COMMENT_PREFIX = re.compile(r'^[\r\t\f\v ]*#')
```

Comment:
`\s` character is for all whitespace, can replace these `\r\t\f\v `
Also, // is also a comment. So:
```suggestion
INDENTATION_PATTERN = re.compile(r'(^\s*(?:-?\s+)?)')
COMMENT_PREFIX = re.compile(r'^[\s]*(#|\/\/)')
```